docs/resources.rst

```diff
@@ -0,0 +1,81 @@
+Resources
+=========
+
+Tutorials
+---------
+
+Django Debugging Tutorial
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Originally presented as an in-person workshop at DjangoCon US 2022, this
+tutorial by **Tim Schilling** covers debugging techniques in Django. Follow
+along independently using the slides and GitHub repository.
+
+* `Read the blog post <https://www.better-simple.com/django/2022/11/18/django-debugging-tutorial/>`__
+* `Follow along with the GitHub repo <https://github.com/tim-schilling/debug-tutorial/>`__
+* Last updated: February 13, 2025.
+* Estimated time to complete: 1-2 hours.
+
+Mastering Django Debug Toolbar: Efficient Debugging and Optimization Techniques
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This tutorial by **Bob Berderbos** provides an in-depth look at effectively
+using Django Debug Toolbar to debug Django applications, covering installation,
+configuration, and practical usage.
+
+* `Watch on YouTube <https://www.youtube.com/watch?v=c5riXBYFxLk>`__
+* Published: May 13, 2023.
+* Duration: 11 minutes.
+
+Talks
+-----
+
+A Related Matter: Optimizing Your Web App by Using Django Debug Toolbar
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Presented at DjangoCon US 2024 by **Christopher Adams**, this talk delves into
+optimizing web applications using Django Debug Toolbar, focusing on SQL query
+analysis and performance improvements.
+
+* `Watch on DjangoTV <https://djangotv.com/videos/djangocon-us/2024/a-related-matter-optimizing-your-webapp-by-using-django-debug-toolbar-with-christopher-adams/>`__
+* `Watch on YouTube <https://www.youtube.com/watch?v=iKn5VWD9l6E>`__
+* Published: December 6, 2024.
+* Duration: 26 minutes.
+
+Fast on My Machine: How to Debug Slow Requests in Production
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Presented at DjangoCon Europe 2024 by **Raphael Michel**, this talk explores
+debugging slow requests in production. While not focused on Django Debug
+Toolbar, it highlights performance issues the tool can help diagnose.
+
+* `Watch on DjangoTV <https://djangotv.com/videos/djangocon-europe/2024/djangocon-europe-2024-fast-on-my-machine-how-to-debug-slow-requests-in-production/>`__
+* `Watch on YouTube <https://www.youtube.com/watch?v=sPO6wDlvKVI>`__
+* Published: July 11, 2024.
+* Duration: 23 minutes.
+
+Criteria for Inclusion
+----------------------
+
+The following criteria ensure that included resources are relevant,
+informative, and valuable to users:
+
+1. Relevance to Django Debug Toolbar
+2. Originality and expertise
+3. Recency and accuracy
+4. Production quality
+5. Practicality and depth
+6. Language and accessibility
```

Comment:
I like the idea of telling people what we're looking at, though I think we may want to reduce this list a tad. If someone is trying to educate others, I'd love to share it with others.

1. Does it at least partially focus on the Django Debug Toolbar?
2. Does the content show a version of Django that is currently supported?
3. What language is the tutorial in and what languages are the captions available as?

What do you think about those reduced questions?